torch/nn/modules/dropout.py

```diff
@@ -54,7 +54,7 @@ def forward(self, input):
 
 
 class Dropout2d(_DropoutNd):
-    r"""Randomly zeroes whole channels of the input tensor.
+    r"""Randomly zeroes whole channels (a channel is a (N, C) pair) of the input tensor.
```

Comment:
This is a bit confusing. A channel is in fact a 2d slice of the last two dimensions.